app/views/leader_sessions/sign_up.html.erb

```diff
@@ -0,0 +1,66 @@
+<h1>Sign up</h1>
+
+<form action="/insert_leader" method="post">
+
+  <div>
+    <label for="email">Email</label>
+    <input id="email" name="email" placeholder="Choose a email..." type="text">
+  </div>
+
+      
+      
+         
+  <!-- Label and input for bio start -->
```

Comment:
We don't want any of these "sentinel" comments in this generator — these are there for Ideas to hook onto when building up an app.